alonet/common/logger.py

```diff
@@ -77,25 +77,31 @@ def log_image(trainer, key, images):
     key: str
         Tag name of the log
     images: list of dict]
-        List of dict with two keys
+        List of dict with
         `image` : torch.Tensor
-        `boxes`: aloscene.BoundingBoxes2D
-    boxes: list, optional
-        List of Dict containing each two keys : name: str, boxes: aloscene.BoundingBoxes2D
+        `boxes`: aloscene.BoundingBoxes2D, optional
+        `masks`: aloscene.Mask, optional
     """
     if isinstance(trainer.logger, WandbLogger):
```

Comment:
Did you handle how this code behave using --log tensorboard ? Its important to support it since its gonna become the default logger at some point.

Reply:
@thibo73800  Done in new commit.